bindings/python/docs/source/changelog.rst

```diff
@@ -0,0 +1,14 @@
+Changelog
+=========
+
+Changes in Version 0.1.0
+------------------------
+
+- Support for patching PyMongo's APIs using :meth:`~pymongoarrow.monkey.patch_all`
```

Comment:
I'd add the main feature: "Support for efficiently converting find and aggregate query results into Arrow/Pandas/Numpy."

Reply:
Done.